docs/src/Masonry.doc.js

```diff
@@ -71,34 +71,38 @@ card(
         name: 'loadItems',
         type: '() => void',
         description:
-          'A callback when the user scrolls and you need to load more items into the grid.',
+          'A callback when the user scrolls and you need to load more items into the grid. Note that `scrollContainer` must be specified.',
```

Comment:
This is great! I remember debugging `loadItems()` for a long time, not realizing that `scrollContainer` had to me specified.